routes/api/contacts.js

```diff
@@ -1,25 +1,81 @@
-const express = require('express')
+const express = require('express');
+const router = express.Router();
+const Joi = require('joi');
 
-const router = express.Router()
+const { HttpError } = require('../../helpers')
+const contact = require('../../models/contacts')
+
+const addSchema = Joi.object({
+  name: Joi.string().required(),
+  email: Joi.string().required(),
+  phone: Joi.string().required(),
+})
 
 router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
+  try {
+    const result = await contact.listContacts();
```

Comment:
This functions should be in separate file called conttroller. Here, in router, they must just be invoked